scheduler.go

```diff
@@ -191,7 +191,7 @@ func (s *Scheduler) calculateMonths(job *Job, lastRun time.Time) time.Duration {
 				nextRun = nextRun.AddDate(0, int(job.interval), daysDifference)
 			}
 		}
-		return s.until(lastRunRoundedMidnight, nextRun)
+		return s.until(lastRun, nextRun)
```

Comment:
@sdw2330976 do you have a time scenario where this fails with `lastRunRoundedMidnight`? We can fake the time for a test, but ideally we could prove the old logic doesn't work with our test